Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "junit-report"
-version = "0.4.2"
+version = "0.4.3"
```

Comment:
I think the change of the public enum `TestResult` makes this an API breaking change so this should be a 0.5.

Reply:
Agreed, let me update it